src/importexport/imagesexport/internal/imagesexportconfiguration.cpp

```diff
@@ -51,6 +52,18 @@ void ImagesExportConfiguration::setExportPdfDpiResolution(int dpi)
     settings()->setSharedValue(EXPORT_PDF_DPI_RESOLUTION_KEY, Val(dpi));
 }
 
+//pdf with transparency
```

Comment:
This comment is not necessary, since the name of the method is already meaningful enough